src/main/java/org/broadinstitute/hellbender/tools/spark/validation/CompareBaseQualitiesSpark.java

```diff
@@ -38,8 +41,27 @@
     @Argument(doc="throw error on diff", shortName = "cd", fullName = "throwOnDiff", optional = true)
     protected boolean throwOnDiff = false;
 
+    /**
+     * Use static quantized quality scores to a given number of levels.
+     */
+    @Advanced
+    @Argument(fullName="static_quantized_quals", shortName = "SQQ", doc = "Use static quantized quality scores to a given number of levels (with -"+ StandardArgumentDefinitions.BQSR_TABLE_SHORT_NAME+ ")", optional=true)
+    public List<Integer> staticQuantizationQuals = new ArrayList<>();
+
+    /**
+     * Round down quantized only works with the static_quantized_quals option.  When roundDown = false, rounding is done in
+     * probability space to the nearest bin.  When roundDown = true, the value is rounded to the nearest bin
+     * that is smaller than the current bin.
+     */
+    @Advanced
+    @Argument(fullName="round_down_quantized", shortName = "RDQ", doc = "Round quals down to nearest quantized qual", optional=true)
```

Comment:
Could you add either a warning or a crash if this is set without the corresponding `static_quantized_quals` being set?